### PR TITLE
Update datadog http_url check URL to v2

### PR DESCRIPTION
**Pull request by dincho**
There is no PT ticket for this. Simple URL update to v2.

Deploy group is also changed because we agreed to add this node to the monitoring some time ago.

### CODE REVIEW

File: ansible/monitoring.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@
         init_config:
         instances:
           - name: API
-            url: "http://localhost:3013/v1/top"
+            url: "http://localhost:3013/v2/top"
```
Comment (michalzajda) [inline ansible/monitoring.yml]: should we keep the `vN` in some central place?

Comment (dincho) [reply ansible/monitoring.yml]: Good point, After we merge #17 I'll rebase and use the variable.

File: deploy.sh [hunk 2]
```diff
@@ -15,5 +15,5 @@ ansible-galaxy install -r requirements.yml
 ansible-playbook environments.yml
 ansible-playbook --limit=epoch setup.yml
 # split monitoring playbook by environment because Ansible group vars are merged otherwise
-ansible-playbook --limit='uat:&epoch:!tester' --extra-vars "datadog_api_key=${datadog_api_key:?}" monitoring.yml
```
Comment (lucafavatella) [inline deploy.sh]: Spurious change?

Comment (dincho) [reply deploy.sh]: I mentioned in the PR description that the tester node is already monitored, this just reflects the current state.

File: ansible/monitoring.yml (hunk 1 above)
Comment (dincho) [reply ansible/monitoring.yml]: see dd59135

File: ansible/monitoring.yml [hunk 3]
```diff
@@ -39,7 +39,7 @@
         init_config:
         instances:
           - name: API
-            url: "http://localhost:3013/v2/top"
+            url: "{{ api_base_uri }}/top"
```
Comment (lucafavatella) [inline ansible/monitoring.yml]: I see defined in https://github.com/aeternity/infrastructure/blob/95c8d0dd616436ad5d8e11b2803d2d787e24b533/ansible/group_vars/epoch/vars.yml#L5

**Pull request merged**